csharp/src/Ice/InputStream.cs

```diff
@@ -132,19 +132,7 @@ public sealed partial class InputStream
         public Encoding Encoding { get; private set; }
 
         /// <summary>The 0-based position (index) in the underlying buffer.</summary>
-        internal int Pos
-        {
-            get => _pos;
-            set
-            {
-                if (value < 0 || value > _buffer.Count)
-                {
-                    throw new ArgumentOutOfRangeException(nameof(Pos),
-                        "the position value is outside the buffer bounds");
-                }
-                _pos = value;
-            }
-        }
+        internal int Pos => _pos;
```

Comment:
We can use auto property now
```suggestion
        internal int Pos { get; private set; }
```

Reply:
Fixed as suggested.